README.md

```diff
@@ -0,0 +1,23 @@
+
+
+# Welcome to Sign
+
+## Brief intro to the repo 
+This is a Client Server Application to upload images to server/ S3 bucket
+
+## Getting Started
+Learn how to setup this project and run it on your local machine.
```

Comment:
this is a simple php application. So you can add the steps how to clone the repo and run on the local apache server